NETWORK/ResetGhostedEntityAlpha.md

```diff
@@ -0,0 +1,17 @@
+---
+ns: NETWORK
+aliases: ["0x17330EBF2F2124A8"]
+---
+## _RESET_GHOSTED_ENTITY_ALPHA
+
+```c
+// 0x17330EBF2F2124A8
+void _RESET_GHOSTED_ENTITY_ALPHA();
+```
+
+Resets the alpha value used by `_SET_LOCAL_PLAYER_AS_GHOST`, `SET_NETWORK_VEHICLE_AS_GHOST`, and `_NETWORK_SET_ENTITY_GHOSTED_WITH_OWNER` back to its default value, 128.
+
+## Parameters
```

Comment:
> Verifying NETWORK/ResetGhostedEntityAlpha.md
> ParseError: No parameter list in "Parameters" heading.

If no parameters are present, there should not be a parameters section.

Same goes for return values. I'm not sure why you added these when renaming?